docs/smart-contracts/overview.md

```diff
@@ -13,37 +13,37 @@ image: /img/og/og-developer-portal.png
 
 Smart contracts are digital agreements defined in code that automate and enforce the terms of a contract without the need for intermediaries, enabling secure and transparent transactions on a blockchain. By leveraging predetermined conditions defined within the smart contract code, the state of a contract can only be updated in a way that follows the rules defined in that contract.
 
-On the Cardano blockchain, the compiled code of smart contracts is stored on, and distributed across the decentralised network. It is not possible to modify the rules of existing smart contract and it is also not possible to de-compile the stored smart contract code back from it's compiled state back in to the original source code.
+On the Cardano blockchain, the compiled code of smart contracts is stored on, and distributed across the decentralised network. It is not possible to modify the rules of an existing smart contract, nor is it possible to decompile the stored smart contract code from its compiled state into the original source code.
 
 ## Introduction
 
 As mentioned in the [general overview](/docs/get-started/), smart contracts on Cardano work a bit differently from how they do on other blockchains. The key to understanding smart contracts is to first understand the [eUTXO](/docs/get-started/technical-concepts/#unspent-transaction-output-utxo) model. 
 
-Smart contracts are more or less just a piece of code that you write to validate the movement of utxos locked in your contract's address. You will lock utxo's at the address of your script and then the utxo's can only ever be spent/moved if your script allows the transaction spending it to do so. 
+Smart contracts are more or less just a piece of code that you write to validate the movement of UTXOs locked in your contract's address. You will lock UTXOs at the address of your script and then the UTXOs can only ever be spent/moved if your script allows the transaction spending it to do so. 
 
 
 ## Conceptual overview
 
 Smart contracts consist of on-chain and off-chain components:
 
-- The on-chain component (validator-script) is a script used to validate that transaction's containing value locked by the script (utxos residing on the script's address) conform to the rules of the contract. Specialized tools and languages are required for creating these scripts.
-- The off-chain component is a script or application that is used to generate transactions that conform to the rules of the contract. These can can be created in almost any language.
+- The on-chain component (validator-script) is a script used to validate that transaction's containing value locked by the script (UTXOs residing on the script's address) conform to the rules of the contract. Specialized tools and languages are required for creating these scripts.
```

Comment:
```suggestion
- The on-chain component (validator-script) is a script used to validate that the transaction's containing value locked by the script (UTXOs residing on the script's address) conforms to the rules of the contract. Specialized tools and languages are required for creating these scripts.
```
@OlofBlomqvist your submission is a complex sentence saying basically "value conform" instead of the proper "value conforms."  But please also double check to ensure that the subject and object are correct before you accept (or modify) this change... I understand the grammar but I have less experience with smart contacts 😜

Reply:
Hmm I see.. what I mean to say is that a validator script makes sure that any **transaction** consuming an UTXO residing on the script's address, **follow the rules**. Not sure if changing "transaction's" to "transactions" would be enough to make that clear or if it should be re-written

Reply:
OK, then this is how to say it what you've said:
> The on-chain component (validator-script) is a script used to validate that the transaction containing the value(s) locked by the script (UTXO(s) residing on the script's address) conforms to the rules of the contract. Specialized tools and languages are required for creating these scripts.

I've also edited this into the diff above so we can compare the before & after and confirm it's accurate.  The simple sentence behind all the complex phrases is: "Each transaction conforms" ... so it's at least grammatically correct. 😇

Reply:
That looks good to me, thanks!  :)